global.json

```diff
@@ -3,4 +3,4 @@
         "src",
         "submodules/FileSystem/src"
     ]
-}
\ No newline at end of file
+}
```

Comment:
I think my vim auto added a new line at the end of the file which was missing. What's the best practice here, new line or no? I will scan the other files for this later.